README.md

```diff
@@ -21,12 +21,8 @@ An example use case of the microservice in payments with FX settlements on a dis
 
 ## Build Instructions
 The following instructions describe how to set up the system.
-  1. Run "start.cmd", one internal and one external service will start.
-  2. Open Postman, or any other application that will simulate the bank system.
-  3. In Postman, click "body" and select "raw".
-  4. Copy the contents of `examples/messages/pacs00800109/example1WithHeader.xml`
-     and paste it into the textbox in Postman.
-  5. Select the "POST" method, enter the URL `localhost:8080/api/v1/pacs/` and
-     click "Send".
-  6. If successfull, you should see a message in the external service's command
-     prompt window that shows the pacs.008 message that "was sent to the bank".
```

Comment:
Maybe these can still be included in the README as instructions how to demo on Windows or something?

Reply:
Instructions on how to demo sound good. I can update the examples with docker compatible examples and add the request demo instructions. As docker works on all platforms I don't see a reason to keep the .cmd script, as it may lead to confusion.